backend/src/emails/accountMultiRegistration/html.pug

```diff
@@ -0,0 +1,22 @@
+doctype html
+html(lang="en")
```

Comment:
Follow issue. The language attribute must be set by the given locale. If not, we will have wrong hyphenation

Reply:
As we send only German emails so far, please set the language to de

Reply:
I have tried to set this, but was not successful til now.
I will try one time again.